### ISSUE TITLE
Display current direct deposit info & offer option to use or change

### ISSUE THREAD

**Issue by vbellissimo** [department-of-veterans-affairs/va.gov-team#77021]
## Issue Description

This task is dependent on #77020

Related Slack thread: https://dsva.slack.com/archives/C06E83RNLQM/p1708959156208089 

- VA is in the process of telling Veterans about a change for those who currently have more than one bank account listed for disability compensation, pension, and education benefit payments
- The same account needs to be used for all of those benefits.
- This is described on VA.gov's Change your [VA Direct Deposit change](https://www.va.gov/change-direct-deposit/)
- "By April 20, 2024, you must have a single bank account for all VA benefit payments"

## User Stories
As a claimant,
I need to know what account I'm already using for education or disability comp payments, and have the option to use that account or change it on my pension claim

As a claimant,
I need to know that if I select a new account, the change won't be made until after a decision is made on my claim, and only if an award is made.

## Acceptance Criteria

- Incorporate necessary copy provided by the Design team from task #77020 relevant to the single Direct Deposit account requirement
- The user will be able to what account they are already using for Education and/or Disability comp payments and have the option to use that account or change it for their Pension submission.
- 
## Figma designs
Updates: [Figma designs](https://www.figma.com/file/9JKK5Eo43uJWEr66JPiebc/Pension-Claim-Form-21P-527EZ?type=design&node-id=2169%3A2603&mode=design&t=USuvgOt0LrFzndak-1) finalized with CAIA

**Comment by gmaciolek** [comment]
Note: I'm waiting to hear back from CAIA re: page URLs. I'll update here when I have an answer.

**Comment by gmaciolek** [comment]
[CAIA responded with feedback on recommended URLs to use](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77464). The conversation is ongoing because I think what they're suggesting is a departure from how we've implemented the page URLs for the entire form.

I suggested the following URLs:

- `/additional-information/direct-deposit`
- `/additional-information/account-information`
- `/additional-information/other-payment-options`


They recommend that we remove `/additional-information` from the URL because it's the section title for a group of questions without any content and would create an empty subdirectory and users would get a 404.

**Comment by aplatt-coforma** [comment]
Changes look good but I did see one line break issue. 
<img width="1622" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNHdSQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--9b786e89e878fada2b775af2f83e28c148ad46f2/Screenshot%202024-03-26%20at%2012.48.43%E2%80%AFPM.png" alt="Screenshot 2024-03-26 at 12.48.43 PM.png" />

@gmaciolek - should it be corrected? If so, I can create a new ticket.

**Comment by aplatt-coforma** [comment]
<img width="584" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNDhSQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--9320d6c169cc9dd767b14864d04758b83655de11/screencapture-staging-va-gov-pension-application-527EZ-additional-information-direct-deposit-2024-03-26-12_41_54.png" alt="screencapture-staging-va-gov-pension-application-527EZ-additional-information-direct-deposit-2024-03-26-12_41_54.png" />
<img width="506" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNU1SQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--85f3af91b516065f69393cc47ebc555bbc806bc6/screencapture-staging-va-gov-pension-application-527EZ-additional-information-other-payment-options-2024-03-26-12_43_52.png" alt="screencapture-staging-va-gov-pension-application-527EZ-additional-information-other-payment-options-2024-03-26-12_43_52.png" />
<img width="455" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNVFSQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--76bd943f4c43cd98a62caaed912eab6ff58ff21d/screencapture-staging-va-gov-pension-application-527EZ-additional-information-account-information-2024-03-26-12_41_40.png" alt="screencapture-staging-va-gov-pension-application-527EZ-additional-information-account-information-2024-03-26-12_41_40.png" />

**Comment by gmaciolek** [comment]
@aplatt-coforma yes, please make a ticket to add the line break in the alert. Thanks for catching that!
